includes/sanitizers/class-amp-gtag-script-sanitizer.php

```diff
@@ -55,5 +55,32 @@ public function sanitize() {
 				ValidationExemption::mark_node_as_px_verified( $script );
 			}
 		}
+
+		/**
+		 * Mark inline gtag events as PX verified attributes.
+		 *
+		 * Such inline events can look like:
+		 *
+		 * onclick="gtag('event','click', { 'event_category':"click", 'event_label':"contactPage" })"
+		 * onsubmit="gtag('event','submit', { 'event_category':"submit", 'event_label':"contactPage" })"
+		 * onkeypress="gtag('event','keypress', { 'event_category':"keypress", 'event_label':"contactPage" })"
+		 */
+		$inline_events = $this->dom->xpath->query(
+			'
+				//@*[
+					substring(name(), 1, 2) = "on"
```

Comment:
Would this be a bit more clear?

```suggestion
					starts-with(name(), "on")
```